submissions/xT0PERx/js-dom/code.js

```diff
@@ -0,0 +1,65 @@
+  },
+];
+
+document.querySelector(".menu-list").addEventListener("click", function (e) {
+  if (e.target && e.target.nodeName === "LI") {
+    switch (e.target.className) {
+      case "menu-item-1":
+        showContent("menu-item-1");
+        break;
+      case "menu-item-2":
+        showContent("menu-item-2");
+        break;
+      case "menu-item-3":
+        showContent("menu-item-3");
+        break;
+      case "menu-item-4":
+        showContent("menu-item-4");
+        break;
+      case "menu-item-5":
+        showContent("menu-item-5");
+        break;
+    }
+  }
+});
+
+function showContent(nameClass) {
+  const element = document.querySelector("#content");
+  while (element.firstChild) {
+    element.removeChild(element.firstChild);
+  }
+  const newDiv = document.createElement("div");
```

Comment:
In some cases no need to create unnecessary div without class. Use document fragment instead [Link](https://developer.mozilla.org/en-US/docs/Web/API/Document/createDocumentFragment)

Reply:
Fixed